packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionTncSection/JurisdictionTncSection.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react';
+import { Text, useBreakpoint } from '@deriv/quill-design';
+import { useModal } from '../../../../../components/ModalProvider';
+import { getStaticUrl } from '../../../../../helpers/urls';
+import { THooks } from '../../../../../types';
+import { companyNamesAndUrls } from '../../../constants';
+import { JurisdictionFootNoteTitle } from '../JurisdictionFootNoteTitle';
+
+type TProps = {
+    isCheckBoxChecked: boolean;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+    setIsCheckBoxChecked: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+/**
+ * Component to show the Terms and Conditions for the selected jurisdiction
+ * @param isCheckBoxChecked
+ * @param setIsCheckBoxChecked
+ * @param selectedCompany
+ * @param selectedJurisdiction
+ * @param param0
+ * @returns
+ */
+
+const JurisdictionTncSection: FC<TProps> = ({ isCheckBoxChecked, selectedJurisdiction, setIsCheckBoxChecked }) => {
+    const { isMobile } = useBreakpoint();
+    const { getModalState } = useModal();
+    const marketType = getModalState('marketType') || 'all';
+    const selectedCompany = companyNamesAndUrls[selectedJurisdiction as keyof typeof companyNamesAndUrls];
+
+    return (
+        <div className='sticky flex flex-col justify-center mt-1500 gap-600 lg:h-2500 bottom-50 bg-system-light-primary-background pt-[15px] px-1000 pb-500 w-screen'>
+            {selectedJurisdiction && (
+                <JurisdictionFootNoteTitle marketType={marketType} selectedJurisdiction={selectedJurisdiction} />
+            )}
+            {selectedJurisdiction && selectedJurisdiction !== 'svg' && (
+                <div className='flex items-center justify-center gap-400'>
+                    <input
+                        checked={isCheckBoxChecked}
+                        className='cursor-pointer'
+                        id='tnc-checkbox'
+                        onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                            setIsCheckBoxChecked(event.target.checked)
+                        }
+                        type='checkbox'
+                    />
+                    <label className='cursor-pointer' htmlFor='tnc-checkbox'>
```

Comment:
Is there anything we can use in quill ?

Reply:
https://quill-design-6v3.pages.dev/?path=/docs/tags-custom--docs

Reply:
We dont have Label but we have Link 
Using Link instead of <a> tag